tex/gloss-korean.ldf

```diff
@@ -0,0 +1,192 @@
+\ProvidesFile{gloss-korean.ldf}[polyglossia: module for Korean]
+
+\PolyglossiaSetup{korean}{
+    script=Hangul,
+    scripttag=hang,
+    language=Korean,
+    langtag=KOR,
+    hyphennames={english,american,usenglish,USenglish},
```

Comment:
I see.  The use case is very reasonable, and there is no better solution for that at the moment (though I can think of two that would be better in the long term).  Let’s leave it at that then.

What do you mean with “enabling British English as well”?